src/plugins/shiftmanager/handler.go

```diff
@@ -0,0 +1,62 @@
+/*
+ * Radon
+ *
+ * Copyright 2018-2020 The Radon Authors.
+ * Code is licensed under the GPLv3.
+ *
+ */
+
+package shiftmanager
+
+import (
+	"github.com/xelabs/go-mysqlstack/sqlparser/depends/sqltypes"
+)
+
+type ShiftStatus int
+
+const (
+	ShiftStatusNone ShiftStatus = iota
+	ShiftStatusMigrating
+	ShiftStatusSuccess
+	ShiftStatusFail
+)
+
+type ShiftType int
+
+const (
+	ShiftTypeNone ShiftType = iota
+	ShiftTypeReshard
+	ShiftTypeRebalance
+)
+
+type ShiftInfo struct {
+	From         string
+	FromUser     string
+	FromPassword string
+	FromDatabase string
+	FromTable    string
+
+	To         string
+	ToUser     string
+	ToPassword string
+	ToDatabase string
+	ToTable    string
+
+	cleanup                bool
+	checksum               bool
+	mysqlDump              string
+	threads                int
+	posBehinds             int
+	waitTimeBeforeChecksum int
+	RadonURL               string
+}
+
+type ShiftManagerHandler interface {
```

Comment:
Shift in the `rebalance` is as follows, NewShift() ->  shift.Start() ->  shift.WaitFinish(). Can you briefly describe how to use it, if we use the `ShiftManagerHandler`.

Reply:
1. get and shift manager
2. shiftMgr.StartAnInstance()
3. WaitInstanceFinishThread()